tests/test_ocs_basic_install.py

```diff
@@ -177,16 +178,17 @@ def create_ebs_volumes(
     """
     aws = AWS(region_name)
     worker_instances = aws.get_instances_by_name_pattern(worker_pattern)
-    for worker in worker_instances:
-        log.info(
-            f"Creating and attaching {size} GB volume to {worker['name']}"
-        )
-        aws.create_volume_and_attach(
-            availability_zone=worker['avz'],
-            instance_id=worker['id'],
-            name=f"{worker['name']}_extra_volume",
-            size=size,
-        )
+    with parallel() as p:
+        for worker in worker_instances:
+            log.info(
+                f"Creating and attaching {size} GB volume to {worker['name']}"
+            )
+            p.spawn(aws.create_volume_and_attach,
```

Comment:
please keep the indent and brackets like it was, following this [coding guidelines](https://github.com/red-hat-storage/ocs-ci/blob/master/docs/coding_guidelines.md)

Reply:
Thanks, updated